tests/testCheckerMain.ml

```diff
@@ -11,6 +12,32 @@ open Error
 let property_test_count = 100
 let qcheck_to_ounit t = OUnit.ounit2_of_ounit1 @@ QCheck_ounit.to_ounit_test t
 
+let test_deploy_function_with_lazy_params_succeeds lazy_params =
+  Ligo.Tezos.reset ();
```

Comment:
I was a bit confused at first about why we had to pass lazy_params when calling DeployFunction instead of just the function bytes. I see that it's used for looking up the function id/bytes, but perhaps a comment here explaining this might help make this more clear.

Reply:
Ah, indeed. The reason I added all these tests was to increase test coverage for `lazyParamsToLazyFunctionId`. That's why these tests are in `testCheckerMain.ml`, but they actually increase test coverage for `checkerEntrypoints.ml`. I'll add a comment about it :+1: